### DISCUSSION TITLE
EV3RTのConsoleに表示される文字について

### DISCUSSION THREAD

**Discussion by akito-udagawa** [toppers/users-forum#94] [ET-Robocon]
![IMG20221015141525](https://user-images.githubusercontent.com/66808497/196354039-44c80791-becd-46b1-a9aa-e813c74c1137.jpg)
![IMG20221015141509](https://user-images.githubusercontent.com/66808497/196354047-8c2cbe38-e045-4c02-bb5e-4fa7ab11d677.jpg)

プログラムが正常に動作しないときコンソールにこのような文が表示されます。
毎回表示されるわけではないので、どのような条件で表示されるのか
また、解決方法を探しています。
ご指摘よろしくお願いします。

【環境】
PC:Windows10
走行体:EV3RT HackEV
開発言語: C++

**Comment by YixiaoLi** [comment]
写真のテキスト自体はアプリ実行を開始する時のログ情報で、特に異常がありません。
毎回このようなログ情報が出力されるべきですが、表示されない時はどんなログが出力されましたか？

**Reply by akito-udagawa** [reply]
写真の表示されないときは何も表示がなくスクロールしても確認できませんでした。

**Comment by hiroakitakada** [comment]
アプリケーションが正常に動作すると，アプリケーションが出力するメッセージで，写真のログ情報が消えてしまう（スクロールアウトされる）が，アプリケーションが正常に動作しない場合は，写真のログ情報が見えているということはないでしょうか？

**Reply by akito-udagawa** [reply]
正常に動作しない場合のみログが見えているのですが、正常に動作した場合はログの出力はスクロールが確認できませんでした。

**Comment by akito-udagawa** [comment]
ログが表示される前後は、以下のようなことがあります。
・キャッシュクリア後
・EV3がよくフリーズする
・シャットダウンが遅い
・操作をしていないとき急に走り出す
